README.rst

```diff
@@ -25,5 +25,5 @@ Disclaimer
 **********
 
 nRF Connect SDK supports development with nRF9160 Cellular IoT devices.
-nRF53 Series devices (which are pre-production) and Thread, Zigbee, and Bluetooth Mesh protocols are supported for development in v1.3.2 for prototyping and evaluation.
+nRF53 Series devices (which are pre-production) and Thread, Zigbee, and Bluetooth Mesh protocols are supported for development in |release| for prototyping and evaluation.
```

Comment:
No - last take on this was to not replace the version, because it won't be resolved in GitHub.
See https://github.com/nrfconnect/sdk-nrf/pull/3082

Reply:
Ok, thanks.